resource-watch/resource-watch
# Hooks linter & Staged lint

## Overview
Adds `react-hooks` to lint React's hooks.
Adds `lint-staged`: it won't allow committing anything **you changed** that doesn't satisfy the linter.

## Testing instructions
Run `yarn` to install dev dependencies. Reset code editor just in case. You should be able to see some warning in hooks from now on.

Regarding `lint-staged`: try to make a commit, it will run a process. If everything is OK, it will proceed as usual, otherwise, it will trigger the things you need to change.

## Pivotal task
Provide the link to the task(s), if any.

---

## Checklist before submitting
